src/encoded/static/components/matrix_sescc_stem_cell.js

```diff
@@ -466,18 +467,31 @@ class MatrixPresentation extends React.Component {
                             const name = formatH9HeaderTitle(d.name);
                             const elementClass = formatPebbleNameToCssClassFriendly(name);
                             const element = document.querySelector(`.${elementClass} th`);
+                            const text = document.querySelector(`.${elementClass} th .subcategory-row-text`);
 
                             if (element) {
-                                element.style.backgroundColor = d.selectedColor;
+                                const cellColor = tintColor(d.selectedColor, 0.1);
+                                const textColor = isLight(cellColor) ? '#000' : '#fff';
+
+                                element.style.backgroundColor = cellColor;
+
+                                if (text) {
+                                    text.style.color = textColor;
+                                }
                             }
                         })
                         .on('mouseout', (d) => {
                             const name = formatH9HeaderTitle(d.name);
                             const elementClass = formatPebbleNameToCssClassFriendly(name);
                             const element = document.querySelector(`.${elementClass} th`);
+                            const text = document.querySelector(`.${elementClass} th .subcategory-row-text`);
 
                             if (element) {
                                 element.style.backgroundColor = 'white';
+
+                                if (text) {
+                                    text.style.color = 'black';
+                                }
```

Comment:
This works so well!

Did you tint the color so that it would be right format? I think you could have tinted with 0 if you didn't want to change the color, but this is also great.

Just as a comment, and it's barely even worth mentioning, I would have used '#000' for 'black' and '#fff' for 'white' here on lines 490 and 493 just because that's what you have on 474, but this super doesn't matter.

Reply:
@emro 
I added a small tint but nothing significant. Are you okay with the small tint? Are you okay if I leave the "black" and "white" instead of "000" and "fff"?

Reply:
Yes! That is all good.